src/components/ProjectsSection/ProjectsSection.js

```diff
@@ -1,14 +1,65 @@
 import React from 'react';
+import styled from 'styled-components';
 import { FormattedMessage } from 'gatsby-plugin-intl';
+import { Image } from '@styled-icons/boxicons-regular/Image';
 
+import { breakPoints } from 'styles/base';
 import SectionHeader from 'styles/elements/SectionHeader';
 import SectionWrapper from 'styles/elements/SectionWrapper';
+import SectionItemWrapper from 'styles/elements/SectionItemWrapper';
+import SectionItemIcon from 'styles/elements/SectionItemIcon';
+
+const ProjectsSectionItemWrapper = styled(SectionItemWrapper)`
+  flex-direction: column;
+  height: 43%;
+  width: 90%;
+
+  @media (min-width: ${breakPoints.desktop}) {
+    flex-direction: ${({ reversed }) => (reversed ? 'row-reverse' : 'row')};
+    width: 60%;
+  }
+`;
+
+const ProjectsSectionItemIcon = styled(SectionItemIcon)`
+  width: 50%;
+  max-width: 150px;
```

Comment:
if you have no set 62.5% for font-sizes and you are using `px` here, I would suggest to use `rem` function from `polished` - btw, awesome package to use with styled-components